package.lisp

```diff
@@ -0,0 +1,3 @@
+(defpackage getpid
+  (:use #:cl)
+  (:export #:save-pid))
```

Comment:
need to export getpid too :)

(re naming: "getpid" without a "-" and "save-pid" with one, is it ok?

Reply:
I pushed a new commit exporting `getpid` also. I must have missed that one ;()

> (re naming: "getpid" without a "-" and "save-pid" with one, is it ok?

`getpid` currently has a `-`. Do you mean that you'd like to rename it to `get-pid`? Not sure I understood...

Reply:
What do you think of renaming `getpid` to `get-process-id` or `get-process-identifier`? 

Username dlowe on ﻿`common-lisp:matrix.org` had suggested this because of tension between a POSIX name and a more lispy version of the name.

Reply:
It's a bit lame, because in all implementations it's named `getpid`… but +1 to rename to `get-pid`.